Src/WitsmlExplorer.Api/Routes.cs

```diff
@@ -67,6 +70,7 @@ public class Routes : CarterModule
             Get("/api/wells", GetAllWells);
             Get("/api/wells/{wellUid}", GetWell);
             Get("/api/wells/{wellUid}/wellbores/{wellboreUid}", GetWellbore);
+            Get("/api/wells/{wellUid}/wellbores/{wellboreUid}/bharuns", GetBhaRun);
```

Comment:
The route does not include a "bhaRunUid" but it calls a method that requires it. The object view needs a route that gets all bhaRuns for a wellbore and this is missing in both bhaRunService and bhaRunQueries.